### DISCUSSION TITLE
Icons not showing on vue with quasar

### DISCUSSION THREAD

**Discussion by ITorres23** [Remix-Design/RemixIcon#465] [General]
I've tried tons of things, from changing manually the width to making a whole new component just for an icon thinking quasar was the problem but nothing. The last time this happened to me on vue with tailwind the solution was just to set the height and the width but now it's not working anymore. Someone have an idea on how to solve this?

**Comment by xiaochunjimmy** [comment]
Have you solved this problem?

**Comment by ITorres23** [comment]
Yes sir! It was quite a long ago, but it was because I FORGOT TO PUT THE IMPORT ON THE APP xD, something simple but hard to see when you focus on other types of error.

**Comment by xiaochunjimmy** [comment]
That's great!
